### PR TITLE
Update "Configuring taxation" docs

### LINKED ISSUE #9351: [Docs] Configuring tax rates needs call to setCategory
http://docs.sylius.com/en/1.1/components_and_bundles/bundles/SyliusTaxationBundle/configuring_taxation.html#configuring-the-tax-rates

### PR DESCRIPTION
**Pull request by pamil**
Fixes #9351.

### CODE REVIEW

File: docs/components_and_bundles/bundles/SyliusTaxationBundle/configuring_taxation.rst [hunk 1]
```diff
@@ -14,24 +14,24 @@ First step is to create a new tax category.
...
-        $factory = $this->container->get('sylius.factory.tax_category');
-        $manager = $this->container->get('sylius.manager.tax_category');
+        $taxCategoryFactory = $this->container->get('sylius.factory.tax_category');
+        $taxCategoryManager = $this->container->get('sylius.manager.tax_category');
 
-        $clothing = $factory
+        $clothingTaxCategory = $taxCategoryFactory
             ->createNew()
             ->setName('Clothing')
             ->setDescription('T-Shirts and this kind of stuff.')
```
Comment (Zales0123) [inline docs/components_and_bundles/bundles/SyliusTaxationBundle/configuring_taxation.rst]: Will it work? We don't have fluent interfaces, so `setName` won't return a tax category entity

File: docs/components_and_bundles/bundles/SyliusTaxationBundle/configuring_taxation.rst [hunk 2]
```diff
@@ -72,27 +72,29 @@ Finally, you have to create appropriate tax rates for each of categories.
...
+        $foodTaxCategory = $taxCategoryRepository->findOneBy(['name' => 'Food']);
 
-        $factory = $this->container->get('sylius.factory.tax_rate');
-        $manager = $this->container->get('sylius.repository.tax_rate');
+        $taxRateFactory = $this->container->get('sylius.factory.tax_rate');
+        $taxRateManager = $this->container->get('sylius.repository.tax_rate');
 
-        $clothingTax = $factory
+        $clothingTaxRate = $taxRateFactory
             ->createNew()
```
Comment (Zales0123) [inline docs/components_and_bundles/bundles/SyliusTaxationBundle/configuring_taxation.rst]: Same here

File: docs/components_and_bundles/bundles/SyliusTaxationBundle/configuring_taxation.rst (hunk 1 above)
Comment (pamil) [reply docs/components_and_bundles/bundles/SyliusTaxationBundle/configuring_taxation.rst]: Good point, it's much more outdated than I thought! :)

Comment (Zales0123) [reply docs/components_and_bundles/bundles/SyliusTaxationBundle/configuring_taxation.rst]: 😄

Comment (pamil) [reply docs/components_and_bundles/bundles/SyliusTaxationBundle/configuring_taxation.rst]: Fixed 🎉

**Comment by Zales0123** [comment]
Thank you, KK! 🚜

**Pull request merged**